sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java

```diff
@@ -440,12 +440,15 @@ public void recordOperation(
                     for (ClientSideRequestStatistics requestStatistics : clientSideRequestStatistics) {
 
                         recordStoreResponseStatistics(
+                            diagnosticsContext,
                             cosmosAsyncClient,
                             requestStatistics.getResponseStatisticsList());
                         recordStoreResponseStatistics(
+                            diagnosticsContext,
                             cosmosAsyncClient,
                             requestStatistics.getSupplementalResponseStatisticsList());
                         recordGatewayStatistics(
+                            diagnosticsContext,
```

Comment:
do we need to do filter on addressResolutionStatistics?

Reply:
Yes! Good catch! Thanks

Reply:
Fixed in next iteration